### PR TITLE
Update temporary file download workflow

### PR DESCRIPTION
**Pull request by ekowidianto**
Instead of temporarily store file download request in the public folder and using SSHFS to connect the downloads folder to NGINX, worker and apps, we now store the files in S3 instead. 

## Changes
- Change `redirect_to` column type in jobs table to text and remove length restriction.
- Enable temporary download file uploading to S3.

### CODE REVIEW

File: lib/autoload/send_file.rb [hunk 1]
```diff
@@ -16,11 +31,16 @@ def self.send_file(file, public_name = File.basename(file))
...
+  def self.send_file_production(file, public_name)
+    current_time = Time.now.to_i
+    s3_key = "downloads/#{current_time}/#{public_name}"
+    object = S3_BUCKET.object(s3_key)
+    File.open(file, 'rb') do |f|
+      object.put(body: f)
+    end
+
+    signer = Aws::S3::Presigner.new
+    url, = signer.presigned_request(:get_object, bucket: S3_BUCKET.name, key: s3_key)
```
Comment (purfectliterature) [inline lib/autoload/send_file.rb:43]: There's no need to use the headers?

File: Gemfile [hunk 2]
```diff
@@ -134,6 +133,7 @@ end
 group :production, :test do
   # Puma will be our app server
   gem 'puma'
+  gem 'aws-sdk-s3'
```
Comment (purfectliterature) [inline Gemfile:136]: Are our tests uploading uploads to S3? If not, I think we can move it down to the `:production` group.

**Review by purfectliterature — APPROVED** [review]
LGTM!

File: lib/autoload/send_file.rb (hunk 1 above)
Comment (ekowidianto) [reply lib/autoload/send_file.rb]: We do not set any headers when generating the url as there is no specific use case I can think of as of now. As such, the headers returned is empty. I just found that there's another method that can be used `presigned_url` that only returns the url.

File: Gemfile (hunk 2 above)
Comment (ekowidianto) [reply Gemfile:139]: Our ci is using it to upload screenshots of failed tests to s3. Moved it to ci instead of test

**Review by purfectliterature — APPROVED** [review]
LGTM, thanks for the replies!

**Pull request merged**